examples/riot-kitchen-sink/src/stories/addon-links.stories.js

```diff
@@ -0,0 +1,15 @@
+import { storiesOf } from '@storybook/riot';
```

Comment:
this story doesn't really use `addon-link`

Reply:
Yes it does not. Any help is welcome. I have tried to refer to LinkTo but it failed

Reply:
`LinkTo` is for React, you need to use `linkTo` or [`withLinks`](https://github.com/storybooks/storybook/tree/master/addons/links#withlinks-decorator)

Reply:
Done